integration-tests/file/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-quartz</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
```

Comment:
Best to run the formatting steps so that all of the pseudo dependencies are added.

Reply:
+1, thanks!